.changeset/sixty-worms-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+Initial commit to open PR (to be removed later)
```

Comment:
Does this changeset need to be removed?

Reply:
Strange, I already remove it (but I understand why it's here again, because of a rebase that didn't remove this commit in the rebased PR). I'll remove it.